lib/bitbot/responder.rb

```diff
@@ -27,6 +27,21 @@ def self.route_for(message)
       false
     end
 
+    def self.access_list_allows?(message)
```

Comment:
Assignment Branch Condition size for access_list_allows? is too high. [16.28/15]